sql/core/src/test/resources/sql-tests/inputs/pivot.sql

```diff
@@ -46,7 +46,7 @@ SELECT * FROM (
   SELECT course, earnings FROM courseSales
 )
 PIVOT (
-  sum(earnings)
+  sum(earnings)LATERAL VIEW
```

Comment:
This update is related to this PR? It seems this change causes the test failure in GA.

Reply:
Mistake,  reverted.